tests/test_unclassified_job.py

```diff
@@ -26,3 +26,17 @@ def test_open_unclassified_failure_log(base_url, selenium):
     treeherder_page.open_next_unclassified_failure()
     logviewer_page = treeherder_page.job_details.open_logviewer()
     assert logviewer_page.is_job_status_visible
+
+
+@pytest.mark.nondestructive
+def test_view_unclassified_jobs(base_url, selenium):
+    page = TreeherderPage(selenium, base_url).open()
+    all_jobs = page.all_jobs
+
+    page.filter_unclassified_jobs()
+    filtered_jobs = page.all_jobs
+
+    job = random.choice(filtered_jobs)
+    unclassified = ['testfailed', 'busted', 'exception']
+    assert any([status for status in unclassified if status in job.title])
```

Comment:
Apologies if this was based on a suggestion by me, but I think we can make this more readable:

```python
assert any(status in job.title for status in unclassified)
```